worlds/terraria/docs/setup_en.md

```diff
@@ -25,7 +37,7 @@ highly recommended to use utility mods and features to speed up gameplay, such a
 ### What is a YAML and why do I need one?
 
 You can see the [basic multiworld setup guide](/tutorial/Archipelago/setup/en) here
-on the Archipelago website to learn about why Archipelago uses YAML files and what they're for.
+on the Archipelago website to learn about what YAML files are used for and how Archipelago uses them.
```

Comment:
This is a voice change again, but both versions' grammar leaves something to be desired. A more correct version:

The [basic multiworld setup guide](/tutorial/Archipelago/setup/en) can be found on Archipelago's website. Among other things, it explains what `.yaml` files are, and how they are used.